### PR TITLE
docs: Adds checklist for env reset into handbooks

### PR DESCRIPTION
**Pull request by JirkaChadima**
Continuation and formalization of our internal discussions about how to proceed with the post-hackathon backwards-incompatible environment reset.

This will conflict with #60, I'll resolve that before merging.

### CODE REVIEW

File: development-handbooks/environment-reset.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+wt-write-api, ...).
+- [ ] Update all Winding Tree *satellite* operations (CRON based hotels etc.).
+
+## Notes
+
+- This might evolve into an automated recurring reset (for example once a month)
+- This might evolve into a release guide for Winding Tree platform as a whole
+- This might not be necessary if we can transparently describe properties
+of our environments
+- This will probably not work for production-like environment
```
Comment (AugustoL) [inline development-handbooks/environment-reset.md]: What about adding another note explaining that the URIs shared by the old index can be reused? Someone might wonder what happens with the data stored off chain..

Comment (JirkaChadima) [reply development-handbooks/environment-reset.md]: That's a good point!

File: development-handbooks/environment-reset.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Checklist
+
+- [ ] Decide a date of the reset (deployment of a breaking change).
+- [ ] Inform community about the planned change (Google Group thread,
+more dev channels in the future). The announcement should contain
+information and links to what has changed and how anyone can update
+their integration to adapt (and how can they get help if needed).
+This should go out a few days before the reset.
+- [ ] Create and deploy new iteration of WTIndex.
```
Comment (AugustoL) [inline development-handbooks/environment-reset.md]: We have a method to set the lif token address that have to be executed after contract creation, https://github.com/windingtree/wt-contracts/blob/master/contracts/WTIndex.sol#L58. I guess it should be mentioned.

Comment (JirkaChadima) [reply development-handbooks/environment-reset.md]: True. But it's in the migration, that's why I've left it out.

Comment (AugustoL) [reply development-handbooks/environment-reset.md]: @hynek-urban nothing for now.

**Pull request merged**